Sitemorse-SCI/readme.txt

```diff
@@ -35,16 +35,15 @@ before it is published and will give you a chance to fix any issues.
 == Installation ==
 
 1. Receive licence key from Sitemorse.
-2. Upload the plugin files to the `/wp-content/plugins/Sitemorse-SCI` directory, or install the plugin through the WordPress plugins screen directly.
+2. Upload the plugin folder to the `/wp-content/plugins/` directory, or install the plugin through the WordPress plugins screen directly.
 3. Activate the plugin through the 'Plugins' screen in WordPress.
 4. Enter the licence key in the Settings->Sitemorse section. Normally other settings can be left with the default values.
 5. For further information please contact your Sitemorse representative.
 
 
 == Testing ===
 Running an assessment on any piece of content will test the full connection. For
-example, from the assessment button whilst editing content or the Sitemorse
-button on the admin bar.
+example, 
```

Comment:
There seems to be something missing here.

Reply:
whoops! Sorry, not sure whats happened there. Adding the original back in